project-planning/development-strategy.md

```diff
@@ -126,43 +138,41 @@ __Assigned to Anna__
 
 ## 5. User story : Clear task
 
-__As a user I should be able to delete the tasks that I want__
-
-__Assigned to Yulia__
+**As a user I should be able to delete the tasks that I want**
 
+**Assigned to Yulia**
 
 ### REPO
 
 - This user story is developed on branch `clear-task`.
 - This branch is merged to `master` branch after completion.
 
 ### Create a deleteItems method
+
 ---
 
 ## 6. User story : Edit task
 
-__As a user I should be able to delete the tasks that I want__
-
-__Assigned to Murrielle__
+**As a user I should be able to delete the tasks that I want**
```

Comment:
I guess Murrielle's story is to edit tasks, not to delete. The code looks very nice to me.

Reply:
Yes that's true. My mistake. It was late when I did the dev-strategy. So you can edit on your branch please. Thank you!

Reply:
@razvanbrb Sure, I will do it)